.travis.yml

```diff
@@ -23,6 +23,7 @@ script:
   - ./vendor/bin/parallel-lint src tests
   - ./vendor/bin/phpunit --coverage-clover build/logs/clover.xml
   - ./vendor/bin/phpcs src --standard=psr2 -sp
+  - ant phpstan
```

Comment:
Is there a way to do this all in .travis.yml without needing ant?

Reply:
Yes I can look for a solution.